### PR TITLE
Revert "[cssom-view] Kill DOMRectList. Fixes https://www.w3.org/Bugs/…

### PR DESCRIPTION
**Pull request by zcorpan**
…Public/show_bug.cgi?id=26200"

This reverts commit ebc0d23a692d703ef48dd634b01db9be433ca6c6.

See https://github.com/w3c/csswg-drafts/issues/1479

Tests: https://github.com/w3c/web-platform-tests/issues/6299

### CODE REVIEW

**Comment by bzbarsky** [comment]
Sounds fine assuming it's a clean revert.  I didn't read the actual diff.

File: cssom-view/Overview.bs [hunk 1]
```diff
@@ -1103,9 +1103,9 @@ partial interface Element {
 
 The <dfn method for=Element>getClientRects()</dfn> method, when invoked, must return the result of the following algorithm:
 
-1. If the element on which it was invoked does not have an associated <a>layout box</a> return an empty sequence and stop this algorithm.
-1. If the element has an associated <a>SVG layout box</a> return a sequence containing a single {{DOMRect}} object that describes the bounding box of the element as defined by the SVG specification, applying the <a>transforms</a> that apply to the element and its ancestors.
-1. Return a sequence containing {{DOMRect}} objects in content order, one for each <a spec=css-break>box fragment</a>, describing its border area (including those with a height or width of zero) with the following constraints:
+1. If the element on which it was invoked does not have an associated <a>layout box</a> return an empty {{DOMRectList}} and stop this algorithm.
```
Comment (annevk) [inline cssom-view/Overview.bs]: object*

**Comment by foolip** [comment]
https://github.com/w3c/web-platform-tests/pull/6299 is already merged, I didn't notice there were two spec changes involved.

**Comment by foolip** [comment]
I went about adding the "object" requested, but don't have write access to this repo. @tabatkins, can you add "object" and merge this?

**Comment by tabatkins** [comment]
Done 👍

**Comment by foolip** [comment]
Thanks!

**Pull request merged**